runWorkflowXToYggHbb.py

```diff
@@ -5,7 +5,8 @@
 
 from detect_mass_points import detect_mass_points
 
-doSyst = False
+doSyst = True
+doSystResBkg = True
```

Comment:
Does the first one imply that the systematics on the signal are used? If so, it could be useful to be more specific in the name of the variable.

Reply:
Changed the name